PyInstaller/building/utils.py

```diff
@@ -657,8 +657,10 @@ def strip_paths_in_code(co, new_filename=None):
         for const_co in co.co_consts
     )
 
+    if hasattr(co, 'co_posonlyargcount'):
+        return co.replace(co_consts=consts)
```

Comment:
The changelog entry says: "use the new ``CodeType.replace()`` method if available." But the code does not test for `.replace`, but for `co_posonlyargcount`. While the later might be an indicator for the former, it is a different test.

Anyway: When using `if is_py38` as a condition, we could spot this as a place where the othercode could be removed in some years when we drop support for Python 3.7.

Reply:
(I know `is_py38` is not yet here. It should get into live on te python-3.8 branch.)

Reply:
This PR is already merged. You can replace hasattr(co, 'co_posonlyargcount') with hasattr(co, 'replace') if you want ;-) In practice, it's basically the same.